erb_lint.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |s|
   s.add_dependency('better_html', '~> 1.0.7')
   s.add_dependency('html_tokenizer')
   s.add_dependency('rubocop', '~> 0.79')
+  s.add_dependency('parser', '>= 2.7.1.4')
```

Comment:
You mentioned:

> Note that this needs the very latest parser to use the new `TreeRewriter#import` feature.

I don't think we have to be concerned with the latest `parser` version not working with the legacy Rubocop paths elsewhere in this PR, but I wanted to double check with you on this. Do you see any issues? Did you manually test to verify this version of the `parser` gem works with Rubocop < 0.87?

Reply:
`parser` doesn't do semantic versioning and instead has a strict "no-incompatibility" policy, so no, it not an issue.